supabase/functions/notification-manager/index.ts

```
@@ -0,0 +1,203 @@
+      case 'user':
+        if (!userId) throw new Error('userId required for user notifications');
+        targetUsers = [userId];
+        break;
+        
+      case 'tenant':
+        if (!tenantId) throw new Error('tenantId required for tenant notifications');
+        const { data: tenantUsers } = await supabase
+          .from('profiles')
+          .select('id')
+          .eq('tenant_id', tenantId)
+          .eq('is_active', true);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)